index.html

```diff
@@ -13,70 +13,110 @@
         background-color: #f7f7f9;
         display: flex;
         flex-direction: column;
-        justify-content: center;
-        align-items: center;
+        /* justify-content: center;
+        align-items: center; */
         height: 100vh;
         overflow: auto;
+        padding-top: 25px;
       }
 
       header {
         text-align: center;
-        padding-top: 100px;
       }
       
       h1 {
         font-size: 2rem;
       }
-            
       h3 {
         font-size: 1.5rem;
         margin-left: 40px;
       }
-      
       p {
         font-size: 1.2rem;
       }
-
       main {
         max-width: 600px;
       }
-      
       .logo {
         height: 125px;
         width: 125px;
         margin-right: auto;
         margin-left: auto;
       }
+      .hide {
+        display: none;
+        padding:0;
+        margin:0;
+      }
+      .center {
+        margin: 0 auto;
+      }
+      .loader {
+        display: inline-block;
+        width: 2rem;
+        height: 2rem;
+        vertical-align: text-bottom;
+        border: .25em solid currentColor;
+        border-right-color: transparent;
+        border-radius: 50%;
+        -webkit-animation:spin 4s linear infinite;
+        -moz-animation:spin 4s linear infinite;
+        animation:spin 4s linear infinite;
+        margin-right:10px;
+      }
+      @-moz-keyframes spin { 100% { -moz-transform: rotate(360deg); } }
+      @-webkit-keyframes spin { 100% { -webkit-transform: rotate(360deg); } }
+      @keyframes spin { 100% { -webkit-transform: rotate(360deg); transform:rotate(360deg); } }
+      .alert {
+        margin-top: 15px;
+      }
     </style>
   </head>
 
   <body>
-    <div id="root"></div>
-    <header>
-      <div>
-        <img id="logo" src="images/web-playground-logo.svg" alt="">
+    <div class = "container">
+      <header>
+        <div class = 'row'>
```

Comment:
The formatting seems super inconsistent between the use of single or double quotes and spacing. Maybe you could could configure Prettier in your IDE so it formats the code automatically?

Reply:
Good idea. Will implement it.